### DISCUSSION TITLE
[Poll] Further encourage users to migrate their assets from Godwoken v0 to v1

### DISCUSSION THREAD

**Discussion by Flouse** [nervosnetwork/godwoken#1023] [Polls]
Dear Godwoken Community,

Godwoken v1 has been running stably for almost a year since 2022-06-22.

In order to further encourage users to migrate their assets from Godwoken [v0](https://v0.gwscan.com/) to [v1](https://v1.gwscan.com/) as soon as possible, a few new operations have been discussed and may be carried out in the coming months.

1. gradually increase the [minimal gas price](https://github.com/godwokenrises/godwoken/discussions/995) of Godwoken v0
2. disable the deposit function of Godwoken v0 at an appropriate time

We understand that this may cause some inconvenience for users still on Godwoken v0, but it’s time to move forward.

Thank you for your understanding and patience during this time.

Sincerely, The Godwoken Team

### Q1. How to withdraw assets from Godwoken v0 to v1?

A: See https://docs.godwoken.io/fastWithdrawal#withdraw-assets-from-godwoken-v0-to-godwoken-v1

TL;DR:
Go to https://bridge.godwoken.io/#/v0/withdrawal and choose `Godwoken v1` as the withdrawal target.

### Q2: Why are you sunsetting Godwoken v0?

A: Godwoken v1 appears to be a more advanced and upgraded version of Godwoken v0, with improvements to EVM-compatibility, security, and code quality. With the release of Godwoken v1, the community appears to be shifting their focus to this newer version, putting more dApps and resources on it. Godwoken v1 needs more community support, it deserves our focus and development effort.

**Comment by Imrall** [comment]
Dzad
